git-pull-request

```diff
@@ -37,7 +37,13 @@ remote_ref = '%s/%s' % (remote_origin, remote_branch)
 
 repo_root = repo.git.rev_parse("--show-toplevel")
 repo_url = list(repo.remotes.origin.urls)[0]
-m = re.search(':([^.]+)(\.|$)', repo_url)
```

Comment:
(wasn't totally sure if there are any other patterns that this was safeguarding?)